packages/shorebird_cli/lib/src/command_pipeline/command_pipeline.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:shorebird_cli/src/command_pipeline/command_pipelines.dart';
+
+class CommandPipeline {
+  CommandPipeline({
+    required this.steps,
+    PipelineContext? context,
+  }) : context = context ?? PipelineContext({});
+
+  final PipelineContext context;
+  final List<PipelineStep> steps;
+
+  Future<void> run() async {
+    var context = PipelineContext({});
```

Comment:
Shouldn't we be using the context member variable?

Reply:
We certainly should, yes :)